src/SourceBuild/content/Directory.Build.props

```diff
@@ -159,6 +159,8 @@
   </PropertyGroup>
 
   <PropertyGroup>
+    <!-- Use current machine distro RID if set. Otherwise, fall back to RuntimeInformation.RuntimeIdentifier -->
+    <TargetRid Condition="'$(TargetRid)' == ''">$(__DistroRid)</TargetRid>
     <TargetRid Condition="'$(TargetRid)' == ''">$([System.Runtime.InteropServices.RuntimeInformation]::RuntimeIdentifier)</TargetRid>
 
     <TargetOS Condition="'$(TargetOS)' == '' AND $([MSBuild]::IsOSPlatform('WINDOWS'))">Windows_NT</TargetOS>
```

Comment:
At line 171, the PortableRid is computed, should this be updated to default to the `__PortableTargetOS`?  Also should the `AdditionalRuntimeIdentified` get set  when `PortableBuild == false`?  I thought that is required part of fixing https://github.com/dotnet/source-build/issues/3578.